### PR TITLE
NumForm on Roman numerals

### PR DESCRIPTION
**Pull request by AngledLuffa**

### CODE REVIEW

File: en_ewt-ud-test.conllu [hunk 1]
```diff
@@ -1109,26 +1109,26 @@
...
-23	VI	vi	NUM	CD	NumType=Card	19	conj	18:nummod|19:conj:and	_
+23	VI	VI	NUM	CD	NumForm=Roman|NumType=Card	19	conj	18:nummod|19:conj:and	_
 24	through	through	ADP	IN	_	29	case	29:case	_
-25	VI	vi	NUM	CD	NumType=Card	27	compound	27:compound	SpaceAfter=No
+25	VI	VI	NUM	CD	NumForm=Roman|NumType=Card	27	compound	27:compound	SpaceAfter=No
 26	-	-	PUNCT	HYPH	_	27	punct	27:punct	SpaceAfter=No
 27	D	D	NOUN	NN	Number=Sing	29	compound	29:compound	SpaceAfter=No
 28	-	-	PUNCT	HYPH	_	29	punct	29:punct	SpaceAfter=No
-29	iii	iii	NUM	CD	NumType=Card	23	nmod	23:nmod:through	SpaceAfter=No
+29	iii	III	NUM	CD	NumForm=Roman|NumType=Card	23	nmod	23:nmod:through	SpaceAfter=No
```
Comment (nschneid) [inline en_ewt-ud-test.conllu:1122]: Actually I don't think the lemma should be capitalized. In the text there is a clear semantic distinction between uppercase and lowercase roman numerals. So if we're preserving the roman/decimal distinction in the lemmas, might as well preserve capitalization too.

File: not-to-release/sources/email/enronsent24_01.xml.conllu [hunk 2]
```diff
@@ -153,7 +153,7 @@
 11	revision	revision	NOUN	NN	Number=Sing	8	obj	8:obj	_
 12	concerning	concern	VERB	VBG	VerbForm=Ger	16	case	16:case	_
 13	Section	section	NOUN	NN	Number=Sing	16	compound	16:compound	_
-14	XIII	xiii	NUM	CD	NumType=Card	13	nummod	13:nummod	_
+14	XIII	xiii	NUM	CD	NumForm=Roman|NumType=Card	13	nummod	13:nummod	_
```
Comment (nschneid) [inline not-to-release/sources/email/enronsent24_01.xml.conllu:156]: lemma should be capitalized

Comment (AngledLuffa) [reply not-to-release/sources/email/enronsent24_01.xml.conllu]: Alright, updated all of the lemmas to match the original text.  Look good?

Comment (nschneid) [reply not-to-release/sources/email/enronsent24_01.xml.conllu]: Yes!

**Comment by nschneid** [comment]
part of #336

**Pull request merged**